gradle.properties

```diff
@@ -12,4 +12,4 @@ gradleDockerComposeVersion=0.17.12
 
 org.gradle.jvmargs=-Xmx6g -XX:MaxMetaspaceSize=2048m
 org.gradle.workers.max=10
-version=2.1.0
```

Comment:
please undo this version bump,  the next-major commit doesn't belong on next-minor